app/services/course/assessment/answer/auto_grading_service.rb

```diff
@@ -0,0 +1,37 @@
+class Course::Assessment::Answer::AutoGradingService
+  class << self
+    # Picks the grader for the given grading, then grades into the given
+    # +Course::Assessment::Answer::AutoGrading+ object.
+    #
+    # @param [Course::Assessment::Answer::AutoGrading] auto_grading The object to store grading
+    #   results in.
+    def grade(auto_grading)
+      pick_grader(auto_grading.answer.question).grade(auto_grading)
+    end
+
+    private
+
+    # Picks the grader to use for the given question.
+    #
+    # @param [Course::Assessment::Question] question The question that the needs to be graded.
+    # @return [Course::Assessment::Answer::AnswerAutoGraderService] The service object that can
+    #   grade this question.
+    def pick_grader(question)
```

Comment:
Unused method argument - `question`. If it's necessary, use `_` or `_question` as an argument name to indicate that it won't be used. You can also write as `pick_grader(*)` if you want the method to accept any arguments but don't care about them.